src/java/org/apache/cassandra/cql3/statements/AlterKeyspaceStatement.java

```diff
@@ -120,11 +122,12 @@ private void validateRFChanges(KeyspaceMetadata ksm, KeyspaceParams params)
 
     private void validateNoRangeMovements()
     {
-        if (allow_alter_rf_during_range_movement || Gossiper.instance == null)
+        if (allow_alter_rf_during_range_movement)
```

Comment:
Gossiper.instance is null in some unit tests which is why I added it. Seems like this would cause issues?

Reply:
Huh I guess not.